lms/djangoapps/bookmarks/services.py

```diff
@@ -79,6 +122,8 @@ def unset_bookmarked(self, usage_key):
         Returns:
             Bool indicating whether the bookmark was removed.
         """
+        self._clear_bookmarks_cache(usage_key.course_key)
```

Comment:
Why wipe the whole bookmark cache, rather than just overwriting the single key that the user is changing?